internal/pkg/cli/cmd/cmd.go

```diff
@@ -66,7 +65,7 @@ type Cmd = cobra.Command
 type RootCommand struct {
 	*Cmd
 	Options   *options.Options
-	Logger    *zap.SugaredLogger
+	Logger    log.Logger
```

Comment:
Commit `Use Logger/DebugLogger interface`

----------------

Nahradenie `*zap.SugaredLogger` -> za interface `log.Logger` na vsetkych miestach.
A nahradenie `*logger.DebugLogger` -> `logger.DebugLogger` (bez hviezdicky), pretoze uz to nie je struktura ale interface.